src/ui/dashboard.py

```
@@ -0,0 +1,74 @@
+from fastapi import FastAPI
+from pydantic import BaseModel
+import yaml
+
+from core.engine import TradingEngine
+from backtest.simulator import BacktestSimulator, _default_strategy
+from execution import PaperTrader, BinanceTrader
+
+app = FastAPI(title="CryptoTrader Dashboard")
+
+
+class RunRequest(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)